tests/e2e/vmc_create_gc.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"os"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	ghttp "github.com/onsi/gomega/ghttp"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	gcManifest = "testing-manifests/tkg/"
+)
+
+var _ = ginkgo.Describe("Create GC", func() {
+	f := framework.NewDefaultFramework("create-gc")
+	var server *ghttp.Server
+	var pvclaim *v1.PersistentVolumeClaim
+	var statusCode int
+
+	ginkgo.BeforeEach(func() {
+
+		bootstrap()
+		_ = getNamespaceToRunTests(f)
+		server = ghttp.NewServer()
+		os.Setenv("SOME_ADDR", server.Addr())
+
+		server.AppendHandlers(
+			ghttp.CombineHandlers(
+				ghttp.VerifyRequest("POST", "/path/to/somepage/VOLUME"),
+				ghttp.RespondWithJSONEncodedPtr(&statusCode, &pvclaim),
+			),
+		)
+	})
+
+	ginkgo.AfterEach(func() {
+		server.Close()
+	})
+
+	/*
+		Test to Create TKC using devops user
+		Steps
+			1.	Get WCP session id with devops user
+			2.	Create TKC with the session id from step 1
+			3.	Verify newly created TKC is up and running
+	*/
+
+	ginkgo.It("[vmc] Create GC using devops user", func() {
+
+		ginkgo.By("Get WCP session id")
+		wcpToken := getWCPSessionId(vmcWcpHost, e2eVSphere.Config.Global.VmcDevopsUser,
+			e2eVSphere.Config.Global.VmcDevopsPassword)
+		framework.Logf("vmcWcpHost %s", vmcWcpHost)
+
+		ginkgo.By("Creating Guest Cluster with Devops User")
+		createGC(vmcWcpHost, wcpToken)
+		ginkgo.By("Validate the Guest Cluster is up and running")
+		gcName := "test-cluster-e2e-script-1"
```

Comment:
use some combination of timestamp and random numbers to keep it unique

Reply:
you are creating it new for existing tests or new tests?

Reply:
As we discussed Ill be using it for existing tests

Reply:
is this declared under a constant? if not replace all occurrences of this string with a constant. Then we can see how to use a better name.